undeploy_closed_branches.py

```diff
@@ -31,6 +31,8 @@ def deployed_features():
 		capture_output=True
 	).stdout
 	apps = json.loads(apps_json)
+	if 'resources' not in apps:
+		print("No apps found in specified space ID, you may be logged into the wrong cloud environment!")
```

Comment:
Not the only reason app might not be found, but a reasonable assumption for now.